.changeset/fair-steaks-kneel.md

```diff
@@ -0,0 +1,7 @@
+---
+'@xstate/machine-extractor': minor
+---
+
+- Support both actors and guards in machine implementations and if both are present, merges them into a single object
```

Comment:
This sounds incorrect. `guards` and `actors` probably shouldn't be merged into a single object. Or have you maybe meant `actors` and `services`?

Reply:
Yeah, it's a typo. It's supposed to be `actors` and `services`.